src/aquarium-optimized/Aquarium.h

```diff
@@ -33,16 +33,26 @@ class Model;
 #include "math.h"
 #endif
 
-enum BACKENDTYPE : short {
-  BACKENDTYPEANGLE,
-  BACKENDTYPEDAWND3D12,
-  BACKENDTYPEDAWNMETAL,
-  BACKENDTYPEDAWNVULKAN,
-  BACKENDTYPED3D12,
-  BACKENDTYPEOPENGL,
-  BACKENDTYPELAST
+enum BACKENDTYPE {
```

Comment:
@hujiajie I added some more comments here, please take another look to see if you like the wording "virtual" and "real" here.

Reply:
I don't have a better idea yet. It's also ok for me without the comments.

Reply:
Maybe the comments here can be "using graphics APIs from third-party libraries" and "using native graphics APIs"?

Reply:
Changed according to offline sync with Jiawei